blog/2024/keycloak-devday-2025-hackathon-announcement.adoc

```diff
@@ -0,0 +1,41 @@
+:title: Keycloak DevDay 2025 Pre-Conf Event Announcement
+:date: 2024-10-21
+:publish: true
+:author: Sebastian Rose
+
+Keycloak Dev Day 2025 is just around the corner, and we would like to invite you to a special pre-event: the Keycloak Hackathon!
+
+== Hackathon: actively help shape Keycloak
+On the day before Dev Day (5th of march), from 10:00 a.m., our hackathon will start, giving you the opportunity to actively contribute to the further development of Keycloak.
+Whether you write code, work on the documentation, improve translations or maintain issues in the issue tracker - everyone
+can take part. The hackathon offers you the opportunity to pitch new ideas and work together in small groups on exciting projects.
+
+== Schedule of the hackathon
+*10:00 am*: Start of the first iteration with a pitch round. Here you can present your ideas and topics, ranging from new features and bug fixes to documentation improvements. The teams start working on the pitched topics. Our goal is to achieve measurable results by the end of the day - be it through code contributions, documentation or other important improvements for the Keycloak community.
```

Comment:
Here you have "10:00 am", later, you just have the time in 24h-format. We should either use 12h-format with am/pm or 24h-format without am/pm.